dev-packages/node-integration-tests/suites/tracing/amqplib/test.ts

```diff
@@ -0,0 +1,52 @@
+import { cleanupChildProcesses, createRunner } from '../../../utils/runner';
+
+jest.setTimeout(30_000);
+
+const EXPECTED_MESSAGE_SPAN_PRODUCER = expect.objectContaining({
+  op: 'message',
+  data: expect.objectContaining({
+    'messaging.system': 'rabbitmq',
+    'otel.kind': 'PRODUCER',
+    'sentry.op': 'message',
+  }),
+  status: 'ok',
+});
+
+const EXPECTED_MESSAGE_SPAN_CONSUMER = expect.objectContaining({
+  op: 'message',
+  data: expect.objectContaining({
+    'messaging.system': 'rabbitmq',
+    'otel.kind': 'CONSUMER',
+    'sentry.op': 'message',
+    'sentry.origin': 'auto.amqplib.otel.consumer',
+  }),
+  status: 'ok',
+});
+
+describe('amqplib auto-instrumentation', () => {
+  afterAll(async () => {
+    cleanupChildProcesses();
+  });
+
+  test('should be able to send and receive messages', done => {
+    createRunner(__dirname, 'scenario-message.ts')
+      .withDockerCompose({
+        workingDirectory: [__dirname],
+        readyMatches: ['Time to start RabbitMQ'],
+      })
+      .expect({
+        transaction: (transaction: any) => {
```

Comment:
```suggestion
        transaction: (transaction: TransactionEvent) => {
```

let's avoid any where possible, this should be easy enough to cast like this?